### PR TITLE
update H2 to v200

### PR DESCRIPTION
**Pull request by jgaleotti**

### CODE REVIEW

File: client-java/controller/src/test/java/org/evomaster/client/java/controller/internal/db/h2/H2SchemaExtractorTest.java [hunk 1]
```diff
@@ -189,7 +192,7 @@ public void testColumnUpperBoundConstraint() throws Exception {
         assertEquals("INTEGER", columnDto.type);
 
         assertEquals(1, fooTable.tableCheckExpressions.size());
-        assertEquals("(AGE_MAX <= 100)", fooTable.tableCheckExpressions.get(0).sqlCheckExpression);
+        assertEquals("(\"AGE_MAX\" <= 100)", fooTable.tableCheckExpressions.get(0).sqlCheckExpression);
```
Comment (arcuri82) [inline client-java/controller/src/test/java/org/evomaster/client/java/controller/internal/db/h2/H2SchemaExtractorTest.java:195]: what was the problem with the escaping of these variables?

File: dbconstraint/src/main/java/org/evomaster/dbconstraint/parser/jsql/JSqlVisitor.java [hunk 2]
```diff
@@ -321,9 +321,22 @@ public void visit(NotEqualsTo notEqualsTo) {
...
+    private static String removeSurroundingQuotes(String str) {
+        return str.substring(1,str.length()-1);
+    }
+
     @Override
     public void visit(Column column) {
         String columnName = column.getColumnName();
+        if (hasSurroundingQuotes(columnName)) {
+            columnName = removeSurroundingQuotes(columnName);
+        }
```
Comment (arcuri82) [inline dbconstraint/src/main/java/org/evomaster/dbconstraint/parser/jsql/JSqlVisitor.java:339]: let's discuss this

File: client-java/controller/src/test/java/org/evomaster/client/java/controller/internal/db/h2/H2SchemaExtractorTest.java (hunk 1 above)
Comment (jgaleotti) [reply client-java/controller/src/test/java/org/evomaster/client/java/controller/internal/db/h2/H2SchemaExtractorTest.java:195]: They started appearing in the latest version of H2.

File: dbconstraint/src/main/java/org/evomaster/dbconstraint/parser/jsql/JSqlVisitor.java (hunk 2 above)
Comment (jgaleotti) [reply dbconstraint/src/main/java/org/evomaster/dbconstraint/parser/jsql/JSqlVisitor.java:339]: Quotes are valid in SQL 
https://stackoverflow.com/questions/2901453/sql-standard-to-escape-column-names
We can conclude that it is something we might get when parsing column names.

Comment (arcuri82) [reply dbconstraint/src/main/java/org/evomaster/dbconstraint/parser/jsql/JSqlVisitor.java:339]: hmmm, looks like thgere was a misunderstanding... these comments should had been added in the code, not just here on GitHub :)

**Review by arcuri82 — CHANGES_REQUESTED** [review]

File: dbconstraint/src/main/java/org/evomaster/dbconstraint/parser/jsql/JSqlVisitor.java (hunk 2 above)
Comment (jgaleotti) [reply dbconstraint/src/main/java/org/evomaster/dbconstraint/parser/jsql/JSqlVisitor.java:339]: Indeed. I will add it to the code explaining the scenario.

**Pull request merged**